iotdb-core/metrics/interface/src/main/java/org/apache/iotdb/metrics/metricsets/system/SystemMetrics.java

```diff
@@ -166,6 +173,48 @@ private void collectSystemMemInfo(AbstractMetricService metricService) {
         a -> osMxBean.getCommittedVirtualMemorySize(),
         SystemTag.NAME.toString(),
         SYSTEM);
+    if (CONFIG.getSystemType() == SystemType.LINUX) {
+      metricService.createAutoGauge(
```

Comment:
so do we need LINUX_MEMORY_COUNT metric?

Reply:
Maybe it's can be flag of how many linux memory we get